src/event.rs

```diff
@@ -22,26 +22,26 @@ use std::{
 ///
 /// These are send transparently to the user library and not handled by routing
 #[derive(Clone, Eq, PartialEq)]
-pub enum ClientEvent {
+pub enum Client {
     /// Inform the user (library) that we are connected to a client
-    ConnectedToClient {
+    ConnectedTo {
         /// Client's endpoint
         peer_addr: SocketAddr,
     },
     /// Inform the user (library) that we are disconnected from a client
-    ConnectionFailureToClient {
+    ConnectionFailureTo {
```

Comment:
This enum seems to mirror the `Event` enum from `quic-p2p` to some extent, so maybe we should use the same naming? So this variant should be just `ConnectionFailure`.